### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-2381)

image

There are lots of static files - they are copied from the existing exred app

### CODE REVIEW

File: core/static/styles/_content.scss [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  border-width: 2px;
+  box-shadow: none;
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 800;
+  overflow: hidden;
+  padding: 10px $spacing-2;
+  text-decoration: none;
+
+  &[disabled="disabled"] {
```
Comment (konradko) [inline core/static/styles/_content.scss]: this highlights in red on GH, relevant?

File: core/static/styles/_content.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  }
+}
+
+.dead {
+  cursor: not-allowed;
+}
+
+
+// MOBILE
+@media (max-width: 420px) {
```
Comment (konradko) [inline core/static/styles/_content.scss]: ditto

File: core/static/styles/_elements.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+
+.link-back-to {
+  @include body-s;
+}
+
+
+// TABLET
+@media (max-width: 767px) {
```
Comment (konradko) [inline core/static/styles/_elements.scss]: ditto

File: core/static/styles/_elements.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+
+// MOBILE
+@media (max-width: 420px) {
```
Comment (konradko) [inline core/static/styles/_elements.scss]: ditto

File: core/static/styles/_forms.scss [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+// Checkboxes + Radio buttons
+.checkbox,
+.radio {
+  @include body-m;
+  margin-bottom: $spacing-4;
+  position: relative;
+  padding: 4px 0 0 45px;
+  -ms-touch-action: manipulation;
+  touch-action: manipulation;
```
Comment (konradko) [inline core/static/styles/_forms.scss]: ditto

File: core/static/styles/_forms.scss [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+}
+.checkbox [type=checkbox]:focus + label::before {
+  -webkit-box-shadow: 0 0 0 3px $focus-yellow;
+  -moz-box-shadow: 0 0 0 3px $focus-yellow;
+  box-shadow: 0 0 0 3px $focus-yellow;
+}
+
+
+// MEDIA QUERIES
+@media (max-width: 767px) {
```
Comment (konradko) [inline core/static/styles/_forms.scss]: +1

File: core/static/styles/_content.scss (hunk 2 above)
Comment (eadpearce) [reply core/static/styles/_content.scss]: this is just a css thing idk why it does that. syntax is perfectly fine

File: core/static/styles/_forms.scss (hunk 5 above)
Comment (eadpearce) [reply core/static/styles/_forms.scss]: another css thing. specifically for touch screens https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action

**Pull request merged**